.github/workflows/dependant-repos.yml

```
@@ -0,0 +1,24 @@
+name: Check that repositories submoduling us will still build after this PR
+
+on:
+  pull_request:
+    branches: [ "master" ]
```

Consider also running this on pushes to `master`. IIUC we can omit the quotes
```suggestion
  pull_request:
    branches: [master]
  push:
    branches: [master]
```

---

I don't think that would do anything since on a push to master there is no PR to comment on.
I think pushes to master should be handled by dependabot trying to uprev mobilecoin on the outside repo.